app/views/thredded/topics/index.html.erb

```diff
@@ -9,10 +9,12 @@
   </div>
 
   <%= content_tag :section, class: 'thredded--main-section thredded--topics', 'data-thredded-topics' => true do %>
+    <% if policy(messageboard).post? %>
```

Comment:
This check is unnecessary because `@new_topic` will only be set in the controller if `policy(messageboard).post?` is `true`.

Please revert the changes in this file.